app/src/main/java/com/example/watpool/services/models/Trips.kt

```diff
@@ -19,6 +19,7 @@ data class Trips (
     val maxPassengers: String = "",
     val isRecurring: Boolean = false,
     val recurringDayOfTheWeek: FirebaseTripsService.DayOfTheWeek = FirebaseTripsService.DayOfTheWeek.SUNDAY,
-    val recurringEndDate: LocalDate = LocalDate.now()
+    val recurringEndDate: LocalDate = LocalDate.now(),
+    val timeString: String = ""
```

Comment:
This should probably be LocalTime. I've merged the firebase changes to support adding a time string